gensim/similarity_learning/evaluation_scripts/WriteUp 25 May.md

```diff
@@ -0,0 +1,87 @@
+# Evaluation of Models
+
+## word2vec averaging evaluation

```

Comment:
Not good description
1. I still have no idea, how I can run benchmark myself. This is many scripts that depend on "hardcoded" stuff.
2. No common table with results of all evaluations and all metrics
3. Do we really need a distinct script for each model? Why not do one common (as we described on a meeting)?

Reply:
Noted.
Will make appropriate changes.

Reply:
Still now re-worked + not updated according to current code-base state.
I'm expect tutorial about "how to reproduce your benchmark", i.e. what I need to install/run for retrieve results